### DISCUSSION TITLE
Docker - Persistent ftpgo.json file

### DISCUSSION THREAD

**Discussion by devishian** [drakkan/sftpgo#1714] [Q&A]
Hi, I'm not able to create with docker (I use portainer) a persistent ftpgo.json file so that I can edit it to personalize my configuration. Can anyone help me?

**Comment by nzx-jsantaana** [comment]
This is one of the main reasons that setting options via environment variables is preferred instead: see https://sftpgo.github.io/latest/env-vars/ . 

Is that an option for you, or is an sftpgo.json file a hard requirement for some reason?
